Alexander_Borisov_L3.py

```diff
@@ -0,0 +1,273 @@
+# HOME WORK 3.1
+
+print('\n----------------\nHOME WORK 3.1\n----------------\n')
+
+print('Program to work with list elements')
+
+hat_list = [1, 2, 3, 4, 5]
+len_list = 0
+i = 0 #index
+
+print('in the following following list', hat_list)
+len_list = len(hat_list)
+i = len_list // 2
+
+print('replace element #', i)
+hat_list[i] = int(input('Enter a new element value (integer):'))
+
+print('Updated list', hat_list, 'contains', len_list, 'elements')
+
+del hat_list[len_list - 1]
+
+print('The list with removed last element:', hat_list)
+
+
+
+# HOME WORK 3.2
+
+print('\n----------------\nHOME WORK 3.2\n----------------\n')
+
+print('Program to work with THE BEATLES')
+
+Beatles = []
+print('\nThe group currently contains', len(Beatles), 'members')
+Beatles.append(input('Help us to create the band. Enter #1 (John Lennon):'))
+print('\nThe group currently contains', len(Beatles), 'members')
+
+Beatles.append(input('Enter #2 (Paul McCartney ):'))
+Beatles.append(input('Enter #3 (George Harrison ):'))
+print('\nThe group contains', len(Beatles), 'members.', 'They are', Beatles)
+
+print('\nAdd #4 (Stu Sutcliffe) and #5 (Pete Best)')
+for i in range (2):
+    Beatles.append(input('Enter #' + str(len(Beatles)+1) + ':'))
+
+print('\nThe group contains', len(Beatles), 'members.', 'They are', Beatles)
+
+print('\nRemove members who left the Group')
+member_died = input('Enter member who has died (Stu Sutcliffe):')
+if (member_died in Beatles):
+    Beatles.remove(member_died)
+else:
+    print('*** probably typo, we will remove', Beatles[3]) 
+    Beatles.pop(3)
+print('Now the group contains', Beatles)
+
+print('\nWe will delete the last member #5 who left the group -', Beatles[-1])
+del Beatles[-1]
+
+Beatles.insert(0, input('Insert a new drummer (Ringo Starr):'))
+
+print('\nThe famous band contained', Beatles)
+
+
+
+# HOME WORK 3.3
+
+print('\n----------------\nHOME WORK 3.3\n----------------\n')
+
+print('SORT sample program')
+
+n = i = 0
+my_list = []
+type_of_sort = ''
+
+n = int(input('Enter how many numbers do u want: (n > 0):'))
+for i in range(n):
+    my_list.append(int(input('Enter next number:'))) 
+
+print('Ok, the list contains the following:', my_list)
+type_of_sort = input('Enter type of sort ("increased" / "reversed" / "decreased"):')
+
+if type_of_sort == 'increased':
+    print('Sort Method =', type_of_sort)
+    my_list.sort()
+elif type_of_sort == 'reversed':
+    print('Sort Method =', type_of_sort)
+    my_list.reverse()
+elif type_of_sort == 'decreased':
+    print('Sort Method =', type_of_sort)
+    my_list.sort()
+    my_list.reverse()
+else:
+    print('Unknown Sort Method =', type_of_sort)
```

Comment:
No no no, sort() and reverse() it is not our approach)) It should be implemented manually! Without of using the standard python functions)

Reply:
Hello tutor, 
sorry, it looks like I missed this point.

Here is an attachment with 
[Alexander_Borisov_L3_3.txt](https://github.com/abarysau/my_py_hw/files/8161529/Alexander_Borisov_L3_3.txt)
my code adapted. I add function  **my_sort**